custom_components/mqtt_vacuum_camera/snapshots/log_files.py

```
@@ -49,84 +47,58 @@ async def async_get_filtered_logs(base_path, directory_path: str, file_name):
     try:
-        # When logger is active.
+        # When logger is active (not at WARNING level 30)
         if (_LOGGER.getEffectiveLevel() > 0) and (_LOGGER.getEffectiveLevel() != 30):
-            await async_get_data(base_path, storage_path, file_name, vacuum_json)
+            await async_get_data(base_path, storage_path, file_name)
             zip_logs(storage_path, file_name)
             if os.path.exists(f"{storage_path}/{file_name}.zip"):
                 source_path = f"{storage_path}/{file_name}.zip"
                 destination_path = f"{hass.config.path()}/www/{file_name}.zip"
                 shutil.copy(source_path, destination_path)
     except Exception as e:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)